src/encord_active/lib/metrics/metric.py

```diff
@@ -122,7 +122,7 @@ def __init__(
         long_description: str,
         metric_type: MetricType,
         data_type: DataType,
-        annotation_type: List[Union[ObjectShape, ClassificationType]],
+        annotation_type: Optional[List[Union[ObjectShape, ClassificationType]]],
```

Comment:
Since this is a list, we should probably just default to an empty one instead of also dealing with `None`s everywhere.

```suggestion
        annotation_type: List[Union[ObjectShape, ClassificationType]] = [],
```